missions/W1/M3/utils/extract/parser.py

```
@@ -0,0 +1,196 @@
+    Parse tbody and return parsed data.
+
+    [IMPORTANT] Only use IMF data in this project
+
+    Return a list of dictionaries. 
+
+    **[WARNING!] thead not used in this project!**   
+    """
+    try:
+        logger.info('테이블 파싱 시작')
+        parsed_data = []
+        for tbody in tbody.find_all('tr'):
```

Ruff (0.8.2)

Loop control variable `tbody` overrides iterable it iterates

(B020)